server/prisma/schema.prisma

```diff
@@ -172,7 +172,7 @@ model event_users {
   rsvp_id        Int
   subscribed     Boolean
   title          String[]
-  user           users            @relation(fields: [user_id], references: [id], onDelete: Cascade, onUpdate: NoAction)
+  user           users            @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

Comment:
there is an issue that when I delete event_user to delete RSVP, way more is deleted with it.

- owner lose their permission when they leave any chapter

```suggestion
  user           users            @relation(fields: [user_id], references: [id], onDelete: Cascade, onUpdate: NoAction)

Reply:
Nvm this, just bad logical statements from my part